### DISCUSSION TITLE
New Location

### DISCUSSION THREAD

**Discussion by panic-Patrick** [mampfes/hacs_waste_collection_schedule#2398] [Q&A]
Hi,
quick question

I don't know if I don't find it, or it's not listed but I can't find "Remondis, Kreis Olpe, Gemeinde Kirchhundem". If is not implemented, could some tell me how to do it?

**Comment by 5ila5** [comment]
Kreis Olpe is supported by the [ICS source ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md) I added dedicated documentation in #2406 you can find it [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/abfuhrtermine_info.md)
